dashboard/app/controllers/api/v1/pd/teacher_attendance_report_controller.rb

```diff
@@ -9,6 +9,11 @@ def index
     @workshops = load_filtered_ended_workshops
 
     report = @workshops.map do |workshop|
+      # TODO(aoby): These workshops are being skipped so as to prevent an
+      # exception from happening when we try to process a user with no account.
+      # Fix this exception and remove this `next` statement.
+      next if workshop.course == Pd::Workshop::COURSE_ADMIN
```

Comment:
To be more safe, this could include `COURSE_COUNSELOR` as well, or even just check [workshop.account_required_for_attendance?](https://github.com/code-dot-org/code-dot-org/blob/a445a3af520980f1181ce0fb67e72bd89daa86e3/dashboard/app/models/pd/workshop.rb#L383-L383) which will cover both.

However I just checked and the next upcoming Counselor workshop isn't for a few weeks.